### ISSUE TITLE
[Bug]: kotlin.UninitializedPropertyAccessException

### ISSUE THREAD

**Issue by davidabreu7** [mallowigi/material-theme-issues#420]
### Describe the bug

Error started appearing after Intellij IDEA update 2023.2.3

```
kotlin.UninitializedPropertyAccessException: lateinit property DESCRIPTORS has not been initialized
	at com.mallowigi.idea.config.MTFileColorsPage.getColorDescriptors(MTFileColorsPage.kt:58)
	at com.intellij.application.options.colors.ColorAndFontOptions.initDescriptions(ColorAndFontOptions.java:586)
	at com.intellij.application.options.colors.ColorAndFontOptions.initPluggedDescriptions(ColorAndFontOptions.java:542)
	at com.intellij.application.options.colors.ColorAndFontOptions.initScheme(ColorAndFontOptions.java:531)
	at com.intellij.application.options.colors.ColorAndFontOptions.initAll(ColorAndFontOptions.java:515)
	at com.intellij.application.options.colors.ColorAndFontOptions.resetImpl(ColorAndFontOptions.java:656)
	at com.intellij.application.options.colors.ColorAndFontOptions.reset(ColorAndFontOptions.java:668)
	at com.mallowigi.idea.MTThemeManager.resetColorScheme(MTThemeManager.kt:872)
	at com.mallowigi.idea.MTLafComponent.appStarted$lambda$1(MTLafComponent.kt:77)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:208)
	at com.intellij.openapi.application.TransactionGuardImpl.access$100(TransactionGuardImpl.java:21)
	at com.intellij.openapi.application.TransactionGuardImpl$1.run(TransactionGuardImpl.java:190)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:861)
	at com.intellij.openapi.application.impl.ApplicationImpl$4.run(ApplicationImpl.java:478)
	at com.intellij.openapi.application.impl.LaterInvocator$1.run(LaterInvocator.java:100)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:79)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:121)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:41)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:792)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:739)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:733)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:761)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:690)
	at com.intellij.ide.IdeEventQueue._dispatchEvent$lambda$10(IdeEventQueue.kt:593)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWithoutImplicitRead(ApplicationImpl.java:1485)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:593)
	at com.intellij.ide.IdeEventQueue.access$_dispatchEvent(IdeEventQueue.kt:67)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:369)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:368)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:787)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:368)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:363)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$1(IdeEventQueue.kt:997)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:105)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:997)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$7(IdeEventQueue.kt:363)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:861)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:405)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
```

### To Reproduce

1. Open the IDE
2. error pops up

### Expected behavior

no error

### Material Theme Version

8.12.2

### IDE

IntelliJ IDEA

### System Info

```shell
IntelliJ IDEA 2023.2.3 (Ultimate Edition)
Build #IU-232.10072.27, built on October 11, 2023
Licensed to ília Digital / David Abreu
Subscription is active until June 18, 2024.
Runtime version: 17.0.8.1+7-b1000.32 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 11.0
GC: G1 Young Generation, G1 Old Generation
Memory: 3072M
Cores: 20
Registry:
    editor.focus.mode.color.light=6b727d
    ide.editor.tab.selection.animation=true
    debugger.new.tool.window.layout=true
    ide.animate.toolwindows=true
    ide.new.editor.tabs.vertical.borders=true
    ide.experimental.ui=true
    ide.balloon.shadow.size=0
    editor.focus.mode.color.dark=6b727d

Non-Bundled Plugins:
    com.jetbrains.space (232.10072.29)
    com.github.copilot (1.3.2.3479)
    org.jetbrains.plugins.go-template (232.9921.89)
    com.redhat.devtools.intellij.telemetry (1.0.0.44)
    com.intellij.kubernetes (232.10072.33)
    com.intellij.guice (232.8660.142)
    com.redhat.devtools.intellij.quarkus (1.29.0.583)
    com.haulmont.jpab (2023.3.5-232)
    Pythonid (232.10072.27)
    izhangzhihao.rainbow.brackets (2023.3.6)
    com.chrisrm.idea.MaterialThemeUI (8.12.2)
    org.sonarlint.idea (9.1.0.75538)
    com.mallowigi (93.1.0)
    zielu.gittoolbox (500.0.21+222)
    mobi.hsz.idea.gitignore (4.5.2)
    aws.toolkit (1.84-232)

Kotlin: 232-1.9.0-IJ10072.27
```

**Comment by mallowigi** [comment]
Update to the latest version

**Comment by davidabreu7** [comment]
@mallowigi when upgrading to Intellij IDEA 2023.2.3 the only option in the marketplace to install is 8.12.2, when downgrading to Intellij IDEA 2023.2.2 the 8.12.5 shows up.
Can you replicate this behavior?

**Comment by mallowigi** [comment]
I'll send a new version 8.12.6

**Comment by BlockyTheDev** [comment]
@mallowigi Is it normal, that my Intellij 2023.2.3 showed the Material Theme UI version v8.12.2 as the newest version until some hours ago and now downgraded to the version v8.10.0? I am asking it here in this closed issue because above the conversation was about the version thing with 8.12.2 and the not shown 8.12.5...

**Comment by mallowigi** [comment]
Yes, I deleted the version 8.12.2 and released a hotfix 8.12.6. But it will take some time to be approved by JetBrains.

In the meantime you can download the version manually at https://plugins.jetbrains.com/plugin/8006-material-theme-ui/versions/stable/422584

--
Elior

**Comment by BlockyTheDev** [comment]
Thanks for the direct link. Stay safe.
